server.js

```diff
@@ -1,16 +1,26 @@
-const express = require('express')
-const app = express()
-const port = process.env.PORT || 3000;
+const express = require('express');
+const app = express();
+const port = 3000;
+const mongoose = require('mongoose');
 
+const uri = 'mongodb+srv://chauhanjanhavi45:Janhavi123@sleep.6wml463.mongodb.net/?retryWrites=true&w=majority';
```

Comment:
Consider moving the database connection logic to a separate file or module. This will make the server.js file cleaner and the code more modular. [important]